plugins/com.google.cloud.tools.eclipse.appengine.newproject/src/com/google/cloud/tools/eclipse/appengine/newproject/AppEngineStandardProjectConfig.java

```diff
@@ -0,0 +1,45 @@
+package com.google.cloud.tools.eclipse.appengine.newproject;
+
+/**
+ * Collects all data needed to create and configure an App Engine Standard Project.
+ */
+class AppEngineStandardProjectConfig {
+
+  private String appEngineProjectId;
+  private String eclipseProjectName;
+  private String packageName;
+  private String eclipseProjectDirectory;
+
+  // todo does builder pattern make more sense here?
```

Comment:
I'm not optimistic about our ability to predict the future. Whatever we do now will change as we add features. For instance, if there's little enough data to fit into a single page, we may make each environment its own wizard, perhaps sharing some widgets between them rather sharing entire pages. I.e. churn will happen, whether we plan for it or not.